ocs_ci/deployment/aws.py

```diff
@@ -186,12 +189,56 @@ class AWSIPI(AWSBase):
     A class to handle AWS IPI specific deployment
     """
 
-    OCPDeployment = IPIOCPDeployment
-
     def __init__(self):
         self.name = self.__class__.__name__
         super(AWSIPI, self).__init__()
 
+    class OCPDeployment(IPIOCPDeployment):
```

Comment:
I think we would want to rename `OCPDeployment`

Reply:
This is how we have it defined in other deployment scenario classes like AWSUPI, AWSUPIFlexy, and IBMCloudIPI.